NotificationSchema.py

```diff
@@ -46,6 +46,13 @@ class Targeting(BaseModel):
     exclude: list[Profile] | None = None
     include: list[Profile] | None = None
 
+    @field_validator('percent_chance')
+    @classmethod
+    def min_max_percent_chance(cls, v: float):
```

Comment:
I believe we can probably use https://docs.pydantic.dev/latest/concepts/fields/#numeric-constraints for this.

Reply:
ooh! nice. making that change now